.ci/jobs/elastic+ems-client+pull-request.yml

```diff
@@ -0,0 +1,18 @@
+---
+- job:
+  name: elastic+ems-client+pull-request
+  display-name: 'elastic / ems-client # pull-request'
+  description: Testing of ems-client pull requests
+  scm:
+  - git:
+      branches:
+      - ${ghprbActualCommit}
+      refspec: +refs/pull/*:refs/remotes/origin/pr/*
+  triggers:
+  - github-pull-request:
+    org-list:
```

Comment:
The `org-list` entry and children need to be indented so the parser can tell they're children of `github-pull-request`. Isn't YAML fun? 😿

Reply:
😫 Let me get out my [YAML ruler](https://www.reddit.com/r/ProgrammerHumor/comments/9fhvyl/writing_yaml/).

Thanks for the help.